tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -892,32 +892,23 @@ SpirvInstruction *SpirvEmitter::loadIfGLValue(const Expr *expr,
         const auto fromType =
```

Comment:
Please remove `QualType boolType = astContext.BoolTy;` in line 890, which is not used anywhere.

Reply:
Done